code/config/lib/Loot/Config/Record.hs

```diff
@@ -133,7 +202,17 @@ type family SubRecsConstrained c k is :: Constraint where
         , SubRecsConstrained c k us
         , SubRecsConstrained c k is
         )
-
+    SubRecsConstrained c k ((l ::+ us) ': is) =
+        ( c (ConfigRec k (SumSelection l : ToBranches us))
+        , SubRecsConstrained c k (SumSelection l : ToBranches us)
```

Comment:
Including `SumSelection` in this line is simply useless, it will be skipped by second pattern-match anyway.

Reply:
Indeed, has been removed